python_modules/dagster/dagster/_core/definitions/metadata/source_code.py

```diff
@@ -218,11 +240,35 @@ def _build_gitlab_url(url: str, branch: str) -> str:
 
 
 @experimental
+@deprecated(breaking_version="1.8", additional_warn_text="Use `link_to_git` instead.")
 def link_to_source_control(
     assets_defs: Sequence[Union["AssetsDefinition", "SourceAsset", "CacheableAssetsDefinition"]],
     source_control_url: str,
     source_control_branch: str,
     repository_root_absolute_path: Union[Path, str],
+) -> Sequence[Union["AssetsDefinition", "SourceAsset", "CacheableAssetsDefinition"]]:
+    caller_file = Path(inspect.stack()[3].filename).absolute()
+    file_anchor_path_in_repository = os.path.relpath(caller_file, repository_root_absolute_path)
+
+    return link_to_git(
+        assets_defs=assets_defs,
+        source_control_url=source_control_url,
+        source_control_branch=source_control_branch,
+        source_control_file_path_mapping=SourceControlFilePathMapping(
+            local_file_anchor=(caller_file),
+            file_anchor_path_in_repository=file_anchor_path_in_repository,
+        ),
+    )
+
+
+@experimental
+def link_to_git(
+    assets_defs: Sequence[Union["AssetsDefinition", "SourceAsset", "CacheableAssetsDefinition"]],
+    source_control_url: str,
```

Comment:
if we're going to use "git" in the title, we should use it for these properties too I think?